### PR TITLE
Fix #3707 - "&hellip;" is displayed in redirect button

### LINKED ISSUE #3707: "&hellip;" is displayed in "Redirecting" button
There is a regression in 1.8.21 - "&hellip;" is displayed in "Redirecting" button in ACP.

image

### PR DESCRIPTION
**Pull request by euantorano**
This should fix #3707, but I haven't tested it yet.

### CODE REVIEW

**Comment by euantorano** [comment]
Good catch, that might be right.

**Comment by JordanMussi** [comment]
Yeah just tested and that works 👍

**Comment by euantorano** [comment]
Change applied, and form change from 1.8.21 applied to `admin/modules/forum/management.php` as it was previously missed there.

File: admin/modules/tools/recount_rebuild.php [hunk 1]
```diff
@@ -426,8 +426,12 @@ function check_proceed($current, $finish, $next_page, $per_page, $name, $name2,
...
 			$(function() { 
 				var button = $(\"#proceed_button\"); 
-				if(button.length > 0) {
-					button.val(\"{$lang->automatically_redirecting}\");
+				if (button.length > 0) {
+					// create a temporary div element to render the text within, unescaping HTML entities
+					var textElement = $('<div/>').html('{$lang->automatically_redirecting}');
+				
+					button.val(textElement.text());
+					button.html(\"{$lang->automatically_redirecting}\");
```
Comment (JordanMussi) [inline admin/modules/tools/recount_rebuild.php]:
```suggestion

```

**Comment by euantorano** [comment]
It might make sense to create a function actually, yeah. I’ll do that tonight.

**Pull request merged**